### ISSUE TITLE
resize not working

### ISSUE THREAD

**Issue by CastixGitHub** [react-native-image-picker/react-native-image-picker#1496]
### Description
I am setting `maxHeight: 1024, saveToPhotos: true,` in options, so I expect the saved photo to be resized
Checking the image dimensions in gallery showed it isn't resized.
also, the response whows original width/height
```
[Tue Nov 17 2020 14:42:48.389]  LOG      oki {"fileName": "rn_image_picker_lib_temp_8ffc48ff-3f7a-4de4-88e1-50b23cb42dd6.jpg", "fileSize": 441685, "height": 3456, "type": "image/jpeg", "uri": "content://com.PanDi.imagepickerprovider/external/Android/data/com.PanDi/files/rn_image_picker_lib_temp_8ffc48ff-3f7a-4de4-88e1-50b23cb42dd6.jpg", "width": 4608}
```

### How to repeat issue and example
* step to reproduce the issue
* ...

### Solution

Resize the image or update readme

### Additional Information

* Image Picker version: `├─ react-native-image-picker@3.0.0-vnext.4`
* React Native version: `├─ react-native@0.63.2`
* Platform: Android, not tested on ios
* Development Operating System: Linux

**Comment by ravirajn22** [comment]
Yes the image saved in the gallery will be in original width and height, but the file you get will be modified.

But I checked the code, there is a bug. For now provide both maxHeight and maxWidth, because if any one is not provided it sends the image with original values.

I will fix in future versions.

**Comment by ziyafenn** [comment]
Would be great to have this fixed to avoid using additional library

**Comment by pnaa** [comment]
I've tested this feature and in fact, passing both parameters maxHeight and maxWidth it will keep the aspect ratio. My worry was if the behavior was stretching the image.

**Comment by ser0ja** [comment]
image dimensions do not depend on the rotation of the camera (screen). always "width" is greater than "height"

**Comment by LucasHimelfarb** [comment]
maxHeight and maxWidth is not resize image, u can not upload images with resolution higher that. As i understand the resize apply when image exceded max values

**Comment by Komal7209** [comment]
I can try to solve this, Just need to link to that file, where this code to this feature has been written, as presently I dont know about whole codebase

And also do correct me if I am wrong, the requirement is just to make image size as per default size of android device, right?

**Comment by Komal7209** [comment]
@CastixGitHub  Could you pls assign this to me  and also pls do provide me with the link to the file where it's code is there as I'm new to the codebase

**Comment by CastixGitHub** [comment]
@Komal7209 I don't have authorship of the project so I can't assign issues.
I don't know the source code enough (and it's been two years since the last time I used this) but I think this is a good starting point for android https://github.com/react-native-image-picker/react-native-image-picker/blob/1394c2385718588610531561ebac2b407862bfe7/android/src/main/java/com/imagepicker/Utils.java#L189

then, as you can see the main issue was a misconception about the difference between the image saved in the gallery and the image pointed by the uri, plus that little bug that required both maxWidth and maxHeight

**Comment by mouhsnimohamed** [comment]
> Yes the image saved in the gallery will be in original width and height, but the file you get will be modified.
> 
> But I checked the code, there is a bug. For now provide both maxHeight and maxWidth, because if any one is not provided it sends the image with original values.
> 
> I will fix in future versions.

Did you fixed it? I have the same issue. if only one parameter is provided (maxWidth OR maxHeight), the image is not resized.
